studio-demo-site-companion.php

```diff
@@ -26,23 +24,27 @@ function studio_companion_admin_notices() {
 			); ?>
 		</p>
 	</div>
-	<style>
-        .studio_notice {
-            border-left-color: #F0B849;
-        }
-
-        .studio_welcome {
-            align-items: center;
-        }
+	<?php
+}
+add_action( 'admin_notices', 'studio_companion_admin_notices' );
 
-		.studio_welcome a {
-			color: #3858E9;
-			text-decoration: none;
-		}
+function studio_companion_admin_enqueue_scripts() {
+	wp_enqueue_style( 'studio-companion', plugin_dir_url( __FILE__ ) . 'assets/style-admin.css' );
+}
+add_action( 'admin_enqueue_scripts', 'studio_companion_admin_enqueue_scripts' );
 
-		.studio_welcome a:hover {
-			text-decoration: underline;
-		}
-	</style>
-	<?php
-}
\ No newline at end of file
+function studio_companion_enqueue_scripts() {
+	wp_enqueue_style( 'studio-companion', plugin_dir_url( __FILE__ ) . 'assets/style.css' );
+	wp_enqueue_script( 'studio-companion', plugin_dir_url( __FILE__ ) . 'assets/index.js', array( ), '1.0', true );
+	wp_localize_script( 'studio-companion', 'studioCompanionNotice', array(
+		'description' => __( "You're previewing a <b>WP Build</b> site." ),
```

Comment:
WP Build? 😀 Can we match the design from the linked issue?

Reply:
Done!, I think I was using a different figma link.

Reply:
Thanks for updating those!